src/libs/actions/EmojiPickerAction.ts

```diff
@@ -0,0 +1,77 @@
+import {ValueOf} from 'type-fest';
+import React from 'react';
+import {View} from 'react-native';
+import CONST from '../../CONST';
+
+type AnchorOrigin = {
+    horizontal: ValueOf<typeof CONST.MODAL.ANCHOR_ORIGIN_HORIZONTAL>;
+    vertical: ValueOf<typeof CONST.MODAL.ANCHOR_ORIGIN_VERTICAL>;
+};
+
+type EmojiPicker = {
+    showEmojiPicker: (onModalHideValue?: () => void, onEmojiSelectedValue?: () => void, emojiPopoverAnchor?: View, anchorOrigin?: AnchorOrigin, onWillShow?: () => void, id?: string) => void;
```

Comment:
What is the purpose of `anchorOrigin` if it always takes exactly same object?

Reply:
`AnchorOrigin` is of type:
```ts
type AnchorOrigin = {
    horizontal: 'left' | 'center' | 'right';
    vertical:  'top' | 'center' | 'bottom';
};
```

Soo, It isn't always the same object 😄

Reply:
ahhhh `CONST.MODAL.ANCHOR_ORIGIN_HORIZONTAL` is actually an object! sorry I missed that 🙇